File: app/src/main/java/com/nearsoft/nearbooks/view/realm/SimpleRealmBaseAdapter.java
package com.nearsoft.nearbooks.view.realm;

import android.content.Context;
import android.view.View;
import android.view.ViewGroup;

import io.realm.RealmBaseAdapter;
import io.realm.RealmObject;
import io.realm.RealmResults;

/**
 * Realm model adapter.
 * Created by epool on 11/20/15.
 */
public class SimpleRealmBaseAdapter<T extends RealmObject> extends RealmBaseAdapter<T> {

    public SimpleRealmBaseAdapter(Context context, RealmResults<T> realmResults, boolean automaticUpdate) {
        super(context, realmResults, automaticUpdate);
    }

    @Override
    public View getView(int position, View convertView, ViewGroup parent) {
        return null;
    }
}

File: app/src/main/java/com/nearsoft/nearbooks/view/adapters/realm/SimpleRealmBaseAdapter.kt
package com.nearsoft.nearbooks.view.adapters.realm

import android.content.Context
import android.view.View
import android.view.ViewGroup

import io.realm.RealmBaseAdapter
import io.realm.RealmObject
import io.realm.RealmResults

/**
 * Realm model adapter.
 * Created by epool on 11/28/15.
 */
class SimpleRealmBaseAdapter<T : RealmObject>(
        context: Context,
        realmResults: RealmResults<T>
) : RealmBaseAdapter<T>(context, realmResults) {

    override fun getView(position: Int, convertView: View, parent: ViewGroup): View? {
        return null
    }

}
